### DISCUSSION TITLE
Time Zone Boundaries

### DISCUSSION THREAD

**Discussion by plaflamme** [OvertureMaps/data#154] [Discussion]
It would be great to add Time Zone information throughout overture maps. Specifically, we'd like to be able to query by lat/lon and obtain the time zone at that location. Presumably, some kind of polygon boundary for time zones would be required for this.

Prior art for extracting time zone polygons from OSM data: https://github.com/evansiroky/timezone-boundary-builder

**Comment by danabauer** [comment]
Thanks for the suggestion, @plaflamme. Our divisions team is currently considering a proposal to add time zone boundaries.
